core/src/epicli/data/common/defaults/configuration/postgresql.yml

```diff
@@ -121,3 +121,16 @@ specification:
       # to have multiple unique filenames per day when dateext option is set
           dateformat -%Y%m%dH%H
       }
+      /var/log/pgbouncer/pgbouncer.log {
```

Comment:
On RHEL there is already existing configuration in `/etc/logrotate.d/pgbouncer` so this one duplicates it.